opentimelineio_contrib/adapters/aaf_adapter/aaf_writer.py

```diff
@@ -75,6 +75,7 @@ def __init__(self, input_otio, aaf_file, **kwargs):
         self._unique_mastermobs = {}
         self._unique_tapemobs = {}
         self._clip_mob_ids_map = _gather_clip_mob_ids(input_otio, **kwargs)
+        self._stackify_nested_groups(input_otio)
```

Comment:
You should make a copy of input_otio before modifying it. Someone calling `write_to_file` would be surprised if the timeline they passed in got modified.